Monika After Story/game/zz_submods.rpy

```diff
@@ -101,6 +101,14 @@ init -991 python in mas_submod_utils:
             except:
                 raise SubmodError("Version number '{0}' is invalid.".format(version))
 
+            #Make sure author and name are proper label names
+            import re
+            prog = re.compile(ur'^[ a-zA-Z_\u00a0-\ufffd][ 0-9a-zA-Z_\u00a0-\ufffd]*$')
```

Comment:
compile is expensive and only needs to be done once. This should be moved to the class's static vars